source/develop/release-management/features/external-network-provider.html.md

```diff
@@ -0,0 +1,248 @@
+---
+title: External Network Providers
+category: feature
+authors: mmirecki
+wiki_category: Feature
+wiki_title: Features/External Network Providers
+wiki_revision_count: 1
+wiki_last_updated: 2016-01-25
+feature_name: External Network Providers
+feature_modules: engine,vdsm
+feature_status: In Development
+---
+
+# External Network Providers
+
+
+## Summary
+
+The purpose of this feature is to enable support for networks not defined by oVirt, but supplied by an external provider.
+
+
+### Owner
+
+*   Feature Owner: Marcin Mirecki: [ mmirecki](User:mmirecki)
+
+<!-- -->
+
+*   Email: <mmirecki@redhat.com>
+
+### Detailed Description
+
+Currently the networks used in oVirt deployments are managed internally by oVirt. Many organizations however, use centralized network management systems to handle all their networking, and would like them to cover their oVirt managed environments. The puprose of this feature is to provide an API over which oVirt can communicate with external network management systems, and use the networks defined in them in provisioned VMs.
```

Comment:
We must mention, and state that we depend, on the years-old external network provider infrastructure. Our intent is to make it easier to use by partners.